# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../mesa/mesa_use_correct_linker.patch.cross
# Copyright (C) 2009 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

diff -Nur Mesa-7.6-orig/src/gallium/winsys/drm/Makefile.template Mesa-7.6/src/gallium/winsys/drm/Makefile.template
--- Mesa-7.6-orig/src/gallium/winsys/drm/Makefile.template	2009-10-04 18:46:37.000000000 +0000
+++ Mesa-7.6/src/gallium/winsys/drm/Makefile.template	2009-10-04 18:47:42.000000000 +0000
@@ -82,7 +82,7 @@
 default: depend symlinks $(TOP)/$(LIB_DIR)/gallium/$(LIBNAME)
 
 $(LIBNAME): $(OBJECTS) $(MESA_MODULES) $(PIPE_DRIVERS) $(WINOBJ) Makefile $(TOP)/src/mesa/drivers/dri/Makefile.template
-	$(MKLIB) -noprefix -o $@ \
+	$(MKLIB) -linker '$(CC)' -noprefix -o $@ \
 		$(OBJECTS) $(PIPE_DRIVERS) \
                 -Wl,--start-group $(MESA_MODULES) -Wl,--end-group \
                  $(WINOBJ) $(DRI_LIB_DEPS) $(DRIVER_EXTRAS)
